test/test_biogeochemistry.jl

```diff
@@ -127,12 +125,17 @@ end
 
 @testset "Biogeochemistry" begin
     @info "Testing biogeochemistry setup..."
-    for bgc in (MinimalDiscreteBiogeochemistry, MinimalContinuousBiogeochemistry)
-        for model in (NonhydrostaticModel, HydrostaticFreeSurfaceModel)
-            for arch in archs
-                @info "Testing $bgc in $model on $arch..."
-                test_biogeochemistry!(arch, bgc, model)
-            end
+    for bgc in (MinimalDiscreteBiogeochemistry, MinimalContinuousBiogeochemistry), 
+        model in (NonhydrostaticModel, HydrostaticFreeSurfaceModel),
+        arch in archs,
+        grid in (RectilinearGrid(arch; size = (2, 2, 2), extent = (2, 2, 2)), 
+                LatitudeLongitudeGrid(arch; size = (5, 5, 5), longitude = (-180, 180), latitude = (-85, 85), z = (-2, 0)),
+                LatitudeLongitudeGrid(arch; topology = (Flat, Flat, Bounded), size = (5, ), z = (-2, 0)),
```

Comment:
I think we drop this from here. A flat lat-lon grid makes no sense. Or how about we change it to 
```suggestion
                LatitudeLongitudeGrid(arch; topology = (Periodic, Flat, Bounded), size = (5, 5), λ = (-30, 50), z = (-2, 0)),
                LatitudeLongitudeGrid(arch; topology = (Flat, Bounded, Bounded), size = (5, 5), φ = (0, 40), z = (-2, 0)),
```